File: JS-Basics/MyFirstProgram/projectsCreation.js
function projectsCreation(input){
    let architectName = input[0];
    let numberOfProjects = input[1];
    let hoursNeeded = numberOfProjects*3;
    let output = "The architect " + architectName + " will need " + hoursNeeded + " hours to complete " + numberOfProjects + " project/s.";
    
    console.log(output);
}

projectsCreation(["George", 4]);

File: Java Basics/test4e/src/ProjectsCreation.java
import java.util.Scanner;

public class ProjectsCreation {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String name = scan.nextLine();
        int numOfProjects = Integer.parseInt(scan.nextLine());

        int hrsPerProject = 3;

        int totalTime = numOfProjects * hrsPerProject;

        System.out.printf("The architect %s will need %d hours to complete %d project/s.", name, totalTime, numOfProjects);
    }
}
